### PR TITLE
Add `hasDRM()` and update typings

### PR DESCRIPTION
**Pull request by xavdid**
Hi there!

This PR adds one function: `hasDRM()` it checks for the existence of the encryption file, and if it's there, the book is protected. I didn't add a test here, but the functionality is tested where I use it, [here](https://github.com/xavdid/epub-wordcount/blob/aaf3cbd7bad80d49adb845aa0673b717440a02dc/__tests__/index.test.ts#L160-L177). 

I also updated your typescript definitions. I was following [this](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html) blueprint. I also updated the triple-slash directive per [these instructions](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#red-flags).

Thanks a bunch!

### CODE REVIEW

File: epub.d.ts [hunk 1]
```diff
@@ -30,37 +30,63 @@
...
-		
-		getImage(id: string, callback: (error: Error, data: Buffer, mimeType: string) => void): void;
-		
-		getFile(id: string, callback: (error: Error, data: Buffer, mimeType: string) => void): void;
-	}
-	
-	export = EPub;
+import { EventEmitter } from 'events'
+
+declare class EPub extends EventEmitter {
```
Comment (xavdid) [inline epub.d.ts]: this was auto-formatted with [prettier](https://prettier.io/), sorry about that. It happens by default in my editor, so it happened at one point while editing. If it's a sticking point, i'm happy to go through and un-format it.

File: epub.d.ts [hunk 2]
```diff
@@ -30,37 +30,63 @@
...
+    callback: (error: Error, data: Buffer, mimeType: string) => void
+  ): void
+
+  hasDRM(): boolean
+}
+
+export = EPub
+
+declare namespace EPub {
+  export interface TocElement {
```
Comment (xavdid) [inline epub.d.ts]: This pattern allows things like:

```ts
import EPub = require('epub')
import { TocElement } from 'epub'

const func = (book: EPub, chapter: TocElement) => ...
```

**Comment by xavdid** [comment]
Looks like there's a test failure. Maybe try updating the `@types/node` dependency: https://github.com/microsoft/TypeScript/issues/32333

**Comment by julien-c** [comment]
LGTM, thanks!

**Comment by julien-c** [comment]
(if you find a way to fix the tests that'd be awesome)

**Comment by xavdid** [comment]
sure, I'll take a look!

**Pull request merged**